2020-07-08 10:39:08,155 - C:/workspace/Python/leetCode/self_learn/logging_test.py[line:38] - ERROR: error
